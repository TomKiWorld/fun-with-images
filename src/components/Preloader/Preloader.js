import React from 'react';
import preloader from './preloader.gif';

/**
 * Site Preloader
 */
const Preloader = () => {
  return (
    <div className='Preloader'>
      <img src={preloader} alt='Loading...'/>
    </div>
  );
}

export default Preloader;
