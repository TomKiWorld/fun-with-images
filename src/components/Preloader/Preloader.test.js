import { shallow } from 'enzyme';
import React from 'react';
import Preloader from './Preloader';

describe('Preloader Component', () => {
  it('Expect to render Preloader Component', () => {
    expect(shallow(<Preloader />)).toMatchSnapshot();
  });
});
