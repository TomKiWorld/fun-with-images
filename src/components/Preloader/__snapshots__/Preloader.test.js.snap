// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader Component Expect to render Preloader Component 1`] = `ShallowWrapper {}`;
